Readme update for adding a role  to billing account for listing project ids (#24)
PrimeOrbit/neptune: docs/onboarding/gcp/README.md

    - Table name: `gcp_billing_export_resource_v1_0127F7_2F14CC_1C4E4E`
    - Tip: The fully qualified table name will always include the word resource. If this is missing, please reconfirm your export type is Detailed usage cost data.
- JSON Key file that was previously created during Service Account creation for `primeorbitsericeaccount`

## Grant Access to Cloud Billing accounts to fetch associated Project Ids

> PrimeOrbit needs <u>READ ONLY</u> access to Billing Account to query Projects Ids.

Reviewer: ```suggestion
PrimeOrbit needs **READ ONLY** access to Billing Account to query Projects Ids.
```

1. Log into the GCP Console.
1. Change Scope to 'billing' level.

> 1. Click on the Billing Account you want to manage.

Reviewer: ```suggestion
1. Click on the billing account you want to manage.
```

> 1. In the left sidebar, go to “Account Management”.

Reviewer: ```suggestion
1. In the left sidebar, go to `Account Management`.
```

> 1. Make sure you are in the right Billing Account level.

Reviewer: ```suggestion
1. Make sure you are in the correct billing account level.
```

> 1. Click on `+ Add principal`<br>

Reviewer: ```suggestion
1. Click on `+ Add principal`.
```

> 1. Enter the 'Service Account' email address shared by PrimeOrbit or the one created above.

Reviewer: ```suggestion
1. Enter the *Service Account* email address shared by PrimeOrbit or the one created above.
```

> 1. In the Select a role dropdown, choose: the role `Billing Account Viewer` from `Billing` Category

Reviewer: ```suggestion
1. In the `Select A Role` dropdown, choose: the role `Billing Account Viewer` from `Billing` Category
```
